docs/iris/example_code/Meteorology/COP_maps.py

```diff
@@ -114,7 +113,7 @@ def main():
         plt.subplot(122)
         plt.title('HadGEM2 A1B-Image Scenario',  fontsize=10)
         contour_result = iplt.contourf(delta_a1b, levels, colors=colors,
-                                       linewidth=0, extend='both')
```

Comment:
Would be interested to see that this is the right thing to do. Will dig a little.